src/app/u30/page.tsx

```diff
@@ -0,0 +1,107 @@
+import giftBigSrc from "@/assets/gift-big.svg";
+import giftSmallSrc from "@/assets/gift-small.svg";
+import { DescriptionItem } from "@/components/landing-page/description-item";
+import { U30Chooser } from "@/components/landing-page/u30/product-chooser";
```

Comment:
are we using this elsewhere? if not i think it would be nicer to move the component to this folder. i feel like `components` should be a library of shared components, while stuff that's used locally should live locally.

Reply:
No, and yes, I generally agree but I had that structure (much) earlier in the project and someone else complained that it's confusing having things spread out across that many top level folders 😅

I leave it for now but am happy to discuss at a later point!